### DISCUSSION TITLE
create a project using 5.1 template.

### DISCUSSION THREAD

**Discussion by neominky** [unoplatform/uno#16700] [Q&A]
### On which page?

https://platform.uno/docs/articles/features/using-skia-gtk.html?tabs=windows%2Cubuntu1804

### What's wrong?

Due to breaking changes to MediaPlayerElement. (https://platform.uno/docs/articles/controls/MediaPlayerElement.html)
explain to use 5.1 templates, but I can't find a way to create a project using 5.1 template.

**Comment by jeromelaban** [comment]
Thanks for the feedback. You can create an app using the 5.1 `dotnet new` templates, as follows:

```bash
dotnet new uninstall uno.templates
dotnet new install uno.templates::5.1.25
```
